src/applications/vaos/containers/ContactInfoPage.jsx

```diff
@@ -70,8 +75,40 @@ const pageKey = 'contactInfo';
 export class ContactInfoPage extends React.Component {
   componentDidMount() {
     this.props.openFormPage(pageKey, uiSchema, initialSchema);
+    this.prefillForm();
```

Comment:
Now that I've done this I realize that the data prefill should likely happen up front and not be part of this component's logic :-/